### ISSUE TITLE
Report not sent though log says it was.

### ISSUE THREAD

**Issue by jcoffland** [FoldingAtHome/fah-client-pub#67]
**Reported by @bb30994 on 26 Jun 2010 15:03 UTC**
See items 5,6 in description of #73

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 26 Jun 2010 18:44 UTC**
One ticket, one issue please.

**Comment by jcoffland** [comment]
**Comment by @bb30994 on 29 Jun 2010 03:08 UTC**
Replying to [jcoffland](comment:1):
> One ticket, one issue please.

GTicket #74 only contains one problem. . . that the REPORT WAS NOT SENT.  Please copy the documentation submitted under #74 to this ticket and fix the problem with the report not being sent which is NOT the same problem.

**Comment by jcoffland** [comment]
**Comment by @bb30994 on 29 Jun 2010 03:11 UTC**
Oops.  I mean copy the logs submitted with #73 to this ticket.  They document both problems.

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 29 Jun 2010 19:36 UTC**
bruce wrote:
5) Onward to something new. Note that in this case, I had no internet connection so the results were NOT sent:

```
13:21:05:Sending unit 00's results
13:21:05:Started thread 1 on PID 3572
13:21:05:Completed 1 unit(s)
```

6: The previous log seems to indicate that the WU was uploaded. After connecting to the internet again and restarting the client, there is no evidence of an actual upload.

```
4:15:26:</config>
14:15:26:Started thread 1 on PID 3432
14:15:26:News: Welcome to Folding@Home
14:15:26:Assigned to work server 143.89.28.70
14:15:26:Requesting new work unit from 143.89.28.70
14:15:27:Downloading 264.80KiB.....done
14:15:32:Unit: 00 State: READY Project: 2972 (Run 25, Clone 31, Gen 12) Core: 78 GRO ID: 0x0000000d0893a1864beb73360000214f
14:15:32:Starting Unit: 00
14:15:32:Downloading core from  http://www.stanford.edu/~pande/Win32/x86/Core_78.fah
14:15:33:Downloading 805.62KiB core package.............done
14:15:36:Valid core signature
14:15:36:Unpacked 2.23MiB to cores/78/FahCore_78.exe
14:15:37:Started thread 2 on PID 3432
T
13:21:05:Clean exit
```

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 15 Jul 2010 19:30 UTC**
```
13:21:05:Sending unit 00's results
13:21:05:Started thread 1 on PID 3572
13:21:05:Completed 1 unit(s)
```
This does not actually indicate that the results were sent.  It never says '''Uploading. . .''' or '''Server responded. . .''' just that it started trying to send a unit.

It doesn't even make any sense to me how this code path could have occurred but this code has change substantially since then so I don't think this can happen anymore.

Anyway, it will need to be tested in the next version.

**Comment by jcoffland** [comment]
**Comment by @jcoffland on 20 Jul 2010 17:56 UTC**
I believe this is fixed in v7.0.8.  Please test and reopen this bug if you find that it is not.
